content/chainguard/chainguard-images/using-the-image-diff-api.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Using the Image Diff API"
+type: "article"
+description: "Learn how to use the Chainguard Images Diff API to compare changes between image versions."
+date: 2023-12-10T08:49:31+00:00
+lastmod: 2023-12-10T08:49:31+00:00
+draft: false
+tags: ["Chainguard Images", "Product", "API"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 450
+toc: true
+---
+
+## Introduction to the Chainguard Images Diff API
+
+The Chainguard Images Diff API generates a JSON formatted list of package additions, removals, and detected CVEs in Chainguard Images. It is intended to provide an authenticated, machine readable, formatted list of changes between two image builds. Combined with the [Tag History API](/chainguard/chainguard-images/using-the-tag-history-api/), you can use the Diff API to compare arbitrary builds of an image over time and programmatically detect changes that might require gating a release or modifying your application code.
+
```

Comment:
The diff API only works with Chainguard Images, right? I might make mention of that somewhere

Reply:
Right, added another Chainguard Images, thanks for the suggestion!